frontend/src/app/shared/components/modal/modal-banner/modal-banner.component.sass

```diff
@@ -0,0 +1,30 @@
+@import '~app/spot/styles/sass/variables'
+
+.op-modal-banner
+  display: grid
+  grid-template-columns: 20px auto max-content
+  grid-gap: 20px
+  align-items: center
+  padding: 10px 20px
+
+  @media only screen and (max-width: 679px)
+    grid-template-columns: auto max-content
+
+  &_warning
+    background: $spot-color-feedback-warning-light
+
+  &_info
+    background: $spot-color-feedback-info-light
+
+  &--text
+    margin: 0
```

Comment:
The `.op-modal-banner--text` as well as the `.op-modal-banner--title` are too big compared to the visuals. Taken from figma, it should be a `caption` (in bold and regular).

You can simply use the `spot-caption` mixin for that. You can also pass `bold` as an argument for the font-weight. The special rules for these two classes should then not be necessary any more.